mods/ra/weapons/smallcaliber.yaml

```diff
@@ -201,16 +201,24 @@ ChainGun:
 		Versus:
 			None: 120
 
-ChainGun.Yak:
+ChainGun.Yak.Right:
 	Inherits: ^HeavyMG
-	ReloadDelay: 3
-	Range: 5c0
-	MinRange: 3c0
+	Burst: 5
```

Comment:
In my opinion the most sensible fix is (probably) to increase ammo to 20, I didn't want to make that call on my own.